test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsControllerSpec.scala

```diff
@@ -2748,6 +2748,51 @@ class AcquisitionDetailsControllerSpec
             }
           )
         }
+
+        "display correct questions on check your answers" when {
+
+          "The question for the acquisition on the cya is correct for all acquisition methods" in {
+            val date = LocalDate.now()
+            List(
+              (AcquisitionMethod.Bought, messages("acquisitionPriceBought.title")),
+              (
+                AcquisitionMethod.Inherited,
+                messages("acquisitionPriceNotBought.title", LocalDateUtils.govDisplayFormat(date))
+              ),
+              (
+                AcquisitionMethod.Gifted,
+                messages("acquisitionPriceNotBought.title", LocalDateUtils.govDisplayFormat(date))
+              ),
+              (
+                AcquisitionMethod.Other("test"),
+                messages("acquisitionPriceNotBought.title", LocalDateUtils.govDisplayFormat(date))
+              )
+            ).foreach {
+              case (method, expectedTitle) =>
+                val nonUkRebasing = CompleteAcquisitionDetailsAnswers(
```

Comment:
it'd be handy to wrap this in a `withClue(s"For $method and $expectedTitle: "){ ... }` so that if it fails we have some idea for which test case it failed on